src/stacked_borrows/mod.rs

```diff
@@ -1153,7 +1153,11 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
         let alloc_extra = this.get_alloc_extra(alloc_id)?;
         let stacks = alloc_extra.stacked_borrows.as_ref().unwrap().borrow();
         for (range, stack) in stacks.stacks.iter_all() {
-            print!("{:?}: [", range);
+            print!("{:?}: ", range);
+            if let Some(bottom) = stack.unknown_bottom() {
+                print!("{:?}..", bottom);
```

Comment:
The meaning of this tag is 'arbitrary many items whose tag is strictly less than `bottom`'. I don't think this way of printing reflects that. So maybe what about `(unknown, < {tag})..`? Or just `(< {tag})`? Annoyingly, `tag` is of the form `<number>`, so the double `<` will look funny either way...

Reply:
🤦 yeah uh for starters I put the `..` on the wrong side of the unknown bottom. What if instead of `[ tag tag tag ]` we did `[ ?..unknown_bottom tag tag tag ]`

I'm not sure I like `<` because that to me says comparison and I feel like ranges are conceptually closer to what's going on here? But also we could be very verbose because this should be rare and print `[ unknown-bottom tag tag tag ]`.

Reply:
`?..unknown_bottom` to me looks like 'some sequence of elements, the last one being `unknown_bottom`'. So I think that is too confusing.

> [ unknown-bottom tag tag tag ].

Yeah verbose might be good, but it should still print the unk-bottom tag. So maybe `[ unknown-bottom(..$tag) $tag $tag ]`?